Add README.md to boa_wasm (#4489)
boa-dev/boa: ffi/wasm/README.md

# @boa-dev/boa_wasm

> WebAssembly bindings for [Boa](https://github.com/boa-dev/boa), an experimental JavaScript lexer, parser and interpreter written in Rust.

Reviewer: ```suggestion
WebAssembly bindings for [Boa](https://github.com/boa-dev/boa), a Javascript engine written in Rust.
```

We're planning to release 1.0 in the near future, so we can remove the "experimental" wording for the next release.

Reviewer: Uhh this wasn't resolved...
